jerry-core/ecma/builtin-objects/ecma-builtin-internal-routines-template.inc.h

```diff
@@ -55,6 +62,13 @@ enum
   PASTE (ECMA_ROUTINE_START_, BUILTIN_UNDERSCORED_ID) = ECMA_BUILTIN_ID__COUNT - 1,
 #define ROUTINE(name, c_function_name, args_number, length_prop_value) \
   ECMA_ROUTINE_ ## name ## c_function_name,
+#define ACCESSOR_VALUE(name, c_getter_func_name, c_setter_func_name, prop_attributes) \
```

Comment:
like ROUTINE, they all be undefined in specific xxx.inc.h (BUILTIN_INC_HEADER_NAME)